drivers/usb/udc/udc_stm32.c

```diff
@@ -1017,20 +1017,12 @@ static void priv_pcd_prepare(const struct device *dev)
 	priv->pcd.Init.dev_endpoints = cfg->num_endpoints;
 	priv->pcd.Init.ep0_mps = cfg->ep0_mps;
 	priv->pcd.Init.speed = PCD_SPEED_FULL;
-
+	priv->pcd.Instance = (PCD_TypeDef *)DT_INST_REG_ADDR(0);
 	/* Per controller/Phy values */
```

Comment:
By the way, inline comments at lines 1016 and 1021 no more applies. They can be removed IMHO.
Alternatively `priv->pcd.Instance` can be moved below inlin comment `/* Per controller/Phy values */`.